tests/installation/system_role.pm

```diff
@@ -1,33 +1,45 @@
 # SUSE's openQA tests
 #
-# Copyright © 2016 SUSE LLC
+# Copyright © 2016-2017 SUSE LLC
 #
 # Copying and distribution of this file, with or without modification,
 # are permitted in any medium without royalty provided the copyright
 # notice and this notice are preserved.  This file is offered as-is,
 # without any warranty.
 
-# Summary: Check system role selection screen. Added in SLE 12 SP2
+# Summary: Check system role selection screen or select system role. Added in SLE 12 SP2
 # Maintainer: Jozef Pupava <jpupava@suse.com>
+# Tags: poo#16650, poo#25850
 
 use strict;
 use base "y2logsstep";
 use testapi;
 use utils 'sle_version_at_least';
 
+
+my %role_hotkey = (
+    default  => 's',    # sles with gnome
```

Comment:
Oh, relying on hot keys on this screen sounds dangerous. One more system role and yast will reshuffle everything. For now we're lucky though as all system roles start with unique and unused letters :)

Reply:
yes, so what? I don't see a problem with that. When the roles change we want to update this. Now we *need* to update it. But otherwise we would have needles so everything is in vain … as is life ;)